### PR TITLE
Docs: Add Missing `.is*` Flags for Shadows

### PR DESCRIPTION
**Pull request by ycw**
fix issue: #23527

### CODE REVIEW

**Review by Mugen87 — APPROVED** [review]

File: docs/api/en/lights/shadows/DirectionalLightShadow.html [hunk 1]
```diff
@@ -82,6 +82,11 @@ <h3>[property:Camera camera]</h3>
 			clipping plane at 0.5 and the [page:OrthographicCamera.far far] clipping plane at 500.
 		</p>
 
+		<h3>[property:Boolean isDirectionalLightShadow]</h3>
+		<p>
+			Read-only flag to check whether a given object is of type [name].
```
Comment (WestLangley) [inline docs/api/en/lights/shadows/DirectionalLightShadow.html:87]: Instead,

    Read-only flag to check if a given object is of type [name].

Comment (ycw) [reply docs/api/en/lights/shadows/DirectionalLightShadow.html:87]: great.

Comment (mrdoob) [reply docs/api/en/lights/shadows/DirectionalLightShadow.html:87]: @ycw are you going to update this PR with @WestLangley suggestion or will you create a new PR?

Comment (ycw) [reply docs/api/en/lights/shadows/DirectionalLightShadow.html:87]: create new PR, including updates for all other tasks :D

**Pull request closed without merging**